src/frontend/delete-github-subscription.ts

```diff
@@ -71,14 +75,24 @@ export default async (req: Request, res: Response): Promise<void> => {
 
       await subscription.destroy();
       await submitProto(action);
+
+      tags.push(`status:202`)
+      statsd.increment('delete-github-subscription.success', tags);
+
       res.sendStatus(202);
     } catch (err) {
+      tags.push(`status:403`)
+      statsd.increment('delete-github-subscription.failure', tags);
+
       res.status(403)
         .json({
           err: `Failed to delete subscription to ${req.body.installationId}. ${err}`,
         });
     }
   } catch (err) {
+    tags.push(`status:400`)
+    statsd.increment('delete-github-subscription.failure', tags);
```

Comment:
what's the standard for these magic strings?

Reply:
What do you mean?

Reply:
`delete-github-subscription.failure`, is there a convention?  What if we wanted to change them? etc

Reply:
Any convention would need to be decided upon by us as a team.

Reply:
Sure, I was just thinking it would be nice to have them all in a single spot.  Not important for now.